### PR TITLE
Update economyGamble.ts

### PR DESCRIPTION
**Pull request by Martin-404**
let martin gamble more then 300k!!!!!!

### CODE REVIEW

**Comment by TheUndo** [comment]
🤔

File: bot/interactions/commands/economy/economyGamble.ts [hunk 1]
```diff
@@ -124,7 +124,7 @@ export const gamble = {
       });
     }
 
-    if (bet > 300_000n) {
+    if (bet > 300_000n && interaction.user.id != "157197838009237504") {
```
Comment (TheUndo) [inline bot/interactions/commands/economy/economyGamble.ts:127]:
```suggestion
    if (bet > 300_000n && interaction.user.id !== "157197838009237504") {
```

**Review by TheUndo — CHANGES_REQUESTED** [review]

File: bot/interactions/commands/economy/economyGamble.ts (hunk 1 above)
Comment (Martin-404) [reply bot/interactions/commands/economy/economyGamble.ts]: task completed

File: bot/interactions/commands/economy/economyGamble.ts [hunk 2]
```diff
@@ -124,7 +124,8 @@ export const gamble = {
       });
     }
 
-    if (bet > 300_000n && interaction.user.id != "157197838009237504") {
+    // only let martin gamble more then limit >:D
```
Comment (TheUndo) [inline bot/interactions/commands/economy/economyGamble.ts:127]: fix grammar
```suggestion
    // Only let Martin gamble more then limit >:D
```

**Review by TheUndo — CHANGES_REQUESTED** [review]

File: bot/interactions/commands/economy/economyGamble.ts (hunk 1 above)
Comment (TheUndo) [reply bot/interactions/commands/economy/economyGamble.ts]: conversion still uresolved

File: bot/interactions/commands/economy/economyGamble.ts (hunk 2 above)
Comment (Martin-404) [reply bot/interactions/commands/economy/economyGamble.ts]: Resolved :neutral_face:

**Comment by Martin-404** [comment]
GYAA

**Pull request merged**